lib/gemstash/cli.rb

```diff
@@ -66,5 +67,20 @@ def status
     def stop
       Gemstash::CLI::Stop.new(self).run
     end
+
+    desc "preload", "Preloads all the gems in your gemstash server from the default upstream"
+    method_option :latest, :type => :boolean, :default => false, :desc =>
+      "Only fetch the latest specs"
+    method_option :server_url, :type => :string, :default => "http://localhost:9292", :desc =>
+      "Gemstash server url"
+    method_option :threads, :type => :numeric, :default => 20, :desc =>
+      "Number of threads to run the fetching job with"
+    method_option :limit, :type => :numeric, :default => nil, :desc =>
+      "Limit for the number of gems to preload"
+    method_option :skip, :type => :numeric, :default => 0, :desc =>
+      "Number of gems to skip when preloading"
+    def preload
```

Comment:
The --config-file makes sense, but I also like to think about this as a client of gemstash. It is bundled there, but it is a one off and is 'kinda' out of the regular execution.

What would be the gain here? having the bind url? unless the server is remote, in which case we would need a specific config file for just one value, unless we build a config file for just this process which seems to be overkill